packages/service/db/migrations/20210720160437_add_document_updates_queue.sql

```diff
@@ -0,0 +1,4 @@
+-- migrate:up
+ALTER TABLE document ADD updates_queue bytea[];
```

Comment:
I like this idea, but it might make more sense as its own table, instead stuffing it into the document table. Then we can slice/dice updates, and we don't have to worry so much about managing concurrent inserts